test/Writer/PhpArrayTest.php

```diff
@@ -116,8 +116,48 @@ public function testWriteConvertsPathToDirWhenWritingBackToFile()
         $this->assertSame($expected, $result);
     }
 
+    public function testRenderWithClassNameScalarsEnabled()
+    {
+        $this->writer->setUseClassNameScalars(true);
+
+        $dummyFqnA = 'ZendTest\Config\Writer\TestAssets\DummyClassA';
```

Comment:
Good point. I thought about not triggering autoload, but ::class does not trigger autoload - I actually know it.